apps/mobile-wallet/src/contexts/SendContext.tsx

```diff
@@ -73,6 +74,7 @@ const SendContext = createContext(initialValues)
 
 export const SendContextProvider = ({ children }: { children: ReactNode }) => {
   const { triggerBiometricsAuthGuard } = useBiometricsAuthGuard()
+  const { triggerFundingPasswordAuthGuard, fundingPasswordModal } = useFundingPassword()
```

Comment:
After many attempts, I was finally satisfied with an API that is similar to the biometrics one (aka `trigger...AuthGuard`. See below for implementation. This is the DRYest solution I can come up with.